nva-datamodel-java/src/test/java/no/unit/nva/model/file/FileModelTest.java

```diff
@@ -293,6 +233,22 @@ private static File.Builder admAgreementBuilder() {
                    .withName(randomString());
     }
 
+    private static String generateNewFile() {
```

Comment:
Checkstyle plus changed publisherAuthority from boolean to publisherVersion enum.